benchmark/tangurnis/Makefile

```diff
@@ -6,10 +6,11 @@ tala_c: tala_c/vel_8.csv tala_c/vel_16.csv tala_c/vel_32.csv tala_c/vel_64.csv t
 
 
 
-%.csv: ../../aspect ba/tan.prm tala/tan.prm tala_c/tan.prm
+%.csv: Makefile code/aspect ba/tan.prm tala/tan.prm tala_c/tan.prm
```

Comment:
here and below, does it make sense to hardcode the path to `aspect`? If you want to keep it variable, maybe define the path to the binary somewhere at the top as a makefile variable.